pkg/nats/proxy/constants.go

```diff
@@ -5,6 +5,7 @@ import "fmt"
 const (
 	ComputeEndpointSubjectPrefix = "node.compute"
 	CallbackSubjectPrefix        = "node.orchestrator"
+	RegistrationSubjectPrefix    = "node.registration"
```

Comment:
looks like `Register` is just one of the methods under this prefix. Does it make sense to call it `node.management`?